iothub/service/src/IotHubClientWebSocket.cs

```diff
@@ -130,18 +130,10 @@ public void Abort()
             this.State = WebSocketState.Aborted;
             try
             {
-#if !NETSTANDARD1_3
                 this.WebSocketStream?.Close(); // Ungraceful close
```

Comment:
just curious: why do we prefer close() over dispose()? dispose() seems to be the cleaner way to signal end-of-life for objects that are disposable.

Reply:
I don't get it. Normally I'd just call Dispose, but it seems someone had a reason that they didn't share here. To be safe, I'll change this to Close and Dispose anywhere we're subsequently assigning it to null.